src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationAPI.js

```diff
@@ -68,14 +71,15 @@ const useAnnotationAPI = ({
     annotationId,
     api,
     currentUser,
+    eventEmitter,
```

Comment:
We should really try to isolate the event pattern as much as possible and not let it leak into other areas. Can we orchestrate this at the common layer for both these hooks?